README.md

```diff
@@ -236,7 +236,7 @@ Please check the AWS Lambda Go same code [here](https://github.com/uber/athenadr
 
 #### Use `athenadriver` Config For Authentication
 
-When environment variable `AWS_SDK_LOAD_CONFIG` is NOT set, you need to pass valid(NOT dummy) `region`, `accessID`, `secretAccessKey` into `athenadriver.NewDefaultConfig()`, in addition to `outputBucket`.
+When environment variable `AWS_SDK_LOAD_CONFIG` is NOT set, you can pass valid (NOT dummy) `region`, `accessID`, `secretAccessKey` into `athenadriver.NewDefaultConfig()`, in addition to `outputBucket`.
```

Comment:
I wonder if it's better to denote here that you MUST pass "outputBucket" and "region" and that the rest are optional?